queries/ruby/highlights.scm

```diff
@@ -16,6 +16,14 @@
   "then"
 ] @keyword
 
+(begin_block) @keyword
+
+(end_block) @keyword
+
+(call
+  method: (identifier) @keyword
+  (#any-of? @keyword "proc" "lambda" "caller" "callcc"))
```

Comment:
These queries get applied before the `@function.call` queries, and are thus overridden. They should work properly if you move them below those queries. Also it seems like maybe the more appropriate capture group would be `@function.builtin`?